README.md

```diff
@@ -0,0 +1,19 @@
+# Halium.org
+
+This is the repository for serving content of
+[Halium Website](https://halium.org/).
+
+## Local development
+
+1. [Install bundler](http://bundler.io/#getting-started)
+2. Run `bundle install`
+3. Run `bundle exec jekyll serve` as [per Jekyll docs](http://jekyllrb.com/docs/quickstart/)
+4. Visit [localhost:4000](http://localhost:4000/)
+
+## File structure
+
+See [Jekyll docs](http://jekyllrb.com/docs/structure/)
+
+## License
+
+All rights reserved to Halium Project.
```

Comment:
Can you please change this to Creative Commons Attribution license? (International version) We have discussed this already in telegram group.. its just I never got around commit it to website.